tests/test-flit-default-pip/pyproject.toml

```diff
@@ -25,5 +26,5 @@ channels = [
 default-non-conda-source = "pip"
 
 [tool.conda-lock.dependencies]
-sqlite = "<3.34"
+sqlite = ">=3.34"
```

Comment:
What's the reason for this? It seems strange to me.

Reply:
I just want to make sure I understand what's going on.

I looked into it, and what's happening is that your command is pulling from Anaconda's `defaults` channel, and the first build there of `sqlite` for `osx-arm64` is v3.35.4. We recently added `osx-arm64` as a default platform in #465. Thus the requested environment is indeed not solvable.

I'm happy to update the `sqlite` pins so that they're solvable with the `defaults` channel for `osx-arm64`.